Umbraco-Cloud/Set-Up/Media/index.md

```diff
@@ -2,184 +2,17 @@
-<security>
-  <services>
-    <!--<service name="LocalFileImageService" type="ImageProcessor.Web.Services.LocalFileImageService, ImageProcessor.Web" />-->
-    <service prefix="media/" name="CloudImageService" type="ImageProcessor.Web.Services.CloudImageService, ImageProcessor.Web">
-      <settings>
-        <setting key="MaxBytes" value="8194304"/>
-        <setting key="Timeout" value="30000"/>
-        <setting key="Host" value="https://[your blob account].blob.core.windows.net/media"/>
-      </settings>
-    </service>
-    <service prefix="remote.axd" name="RemoteImageService" type="ImageProcessor.Web.Services.RemoteImageService, ImageProcessor.Web">
-      <settings>
-        <setting key="MaxBytes" value="4194304" />
-        <setting key="Timeout" value="3000" />
-        <setting key="Protocol" value="http" />
-      </settings>
-      <whitelist>
-        <add url="https://[your Azure CDN].vo.msecnd.net/" />
-        <add url="https://[your blob account].blob.core.windows.net/" />
-        <add url="https://[your Umbraco cloud site].s1.umbraco.io" />
-        <add url="http://localhost" />
-        <add url="http://127.0.0.1" />
-      </whitelist>
-    </service>
-  </services>
-</security>
-```
-
-Update `~/config/imageprocessor/cache.config` by removing the default “DiskCache” config entry.
-```xml
-<?xml version="1.0" encoding="utf-8"?>
-<caching currentCache="AzureBlobCache">
-  <caches>
-    <cache name="AzureBlobCache" type="ImageProcessor.Web.Plugins.AzureBlobCache.AzureBlobCache, ImageProcessor.Web.Plugins.AzureBlobCache" maxDays="365" memoryMaxMinutes="60" browserMaxDays="7">
-      <settings>
-        <setting key="CachedStorageAccount" value="DefaultEndpointsProtocol=https;AccountName=[myAccountName];AccountKey=[myAccountKey]" />
-        <setting key="CachedBlobContainer" value="cache" />
-        <setting key="UseCachedContainerInUrl" value="true" />
-        <setting key="CachedCDNRoot" value="[CdnRootUrl]" />
-        <setting key="CachedCDNTimeout" value="2000" />
-        <setting key="SourceStorageAccount" value="DefaultEndpointsProtocol=https;AccountName=[myAccountName];AccountKey=[myAccountKey]" />
-        <setting key="SourceBlobContainer" value="media" />
-        <setting key="StreamCachedImage" value="false" />
-      </settings>
-    </cache>
-  </caches>
-</caching>
-```
-The final note here is that setting this up will only make it so new media files are added to Blob Storage, if you already have some media files on your project you should copy the contents of the media folder and upload it all to the media blob you set up. Finally you can delete the media folder locally as it is no longer needed. 
+Setting up Azure Blob Storage requires you to install a few packages and follow a few simple steps which we've outline in throrough guide on how to set it up.
 
+In this article you can also find more information, about when using Azure Blob Storage on your Umbraco Cloud project.
```

Comment:
This sentence doesn't make a lot of sense, not sure what you are trying to say? 🙂